src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyn/BeregningBarnetilsynUtil.kt

```diff
@@ -0,0 +1,68 @@
+package no.nav.familie.ef.sak.beregning.barnetilsyn
+
+import no.nav.familie.ef.sak.felles.dto.Periode
+import java.math.BigDecimal
+import java.math.BigDecimal.ZERO
+import java.time.LocalDate
+import java.time.YearMonth
+
+data class MaxbeløpBarnetilsynSats(val fraOgMedDato: LocalDate,
+                                   val tilOgMedDato: LocalDate,
+                                   val maxbeløp: Map<Int, Int>)
+
+object BeregningBarnetilsynUtil {
+
+    val satserForBarnetilsyn: List<MaxbeløpBarnetilsynSats> =
+            listOf(MaxbeløpBarnetilsynSats(fraOgMedDato = LocalDate.of(2022, 1, 1),
+                                           tilOgMedDato = LocalDate.MAX,
+                                           maxbeløp = mapOf(1 to 4250, 2 to 5545, 3 to 6284)),
+                   MaxbeløpBarnetilsynSats(fraOgMedDato = LocalDate.of(2021, 1, 1),
+                                           tilOgMedDato = YearMonth.of(2021, 12).atEndOfMonth(),
+                                           maxbeløp = mapOf(1 to 4195, 2 to 5474, 3 to 6203)),
+                   MaxbeløpBarnetilsynSats(fraOgMedDato = LocalDate.of(2020, 1, 1),
+                                           tilOgMedDato = YearMonth.of(2020, 12).atEndOfMonth(),
+                                           maxbeløp = mapOf(1 to 4053, 2 to 5289, 3 to 5993))
```

Comment:
Kan hende vi må legge inn `0 to 0` for hver av satsene også.

Reply:
...eller bare returnere 0 i "hent" (evt. kaste feil hvis det ikke er lov å ha periode med 0 barn???)

Reply:
Vi gjør jo et "hack" for antall barn > 3 også.

Reply:
Legger inn 0